### PR TITLE
Tourette Syndrome Trait

### PR DESCRIPTION
**Pull request by Gnomee1337**
## About the PR 
Adds new trait "Tourette Syndrome" which gives the character Tourette's symptoms such as: body jittering, uncontrollable speech/screams and accidentally drop/use items.
_Symptoms have been sped up on video for demonstration. In the final commit have set huger timespans between symptoms._

https://user-images.githubusercontent.com/41423212/209868427-07a26887-3473-46bb-b9a4-ed0aaf4f4ab2.mp4

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: gnome1337
- add: Added new trait "Tourette Syndrome"

### CODE REVIEW

File: Resources/Prototypes/Traits/disabilities.yml [hunk 1]
```diff
@@ -22,3 +22,19 @@
...
+  components:
+    - type: TouretteSyndrome
+      timeBetweenIncidents: 60, 400
+      touretteSymptoms:
+        - Scream
+        - Phrase
+        - DropItemInHand
+        - UseItemInHand
+        - Jittering
+      tourettePhrases: ["Ccloooown", "fuck", "woof-woof", "Grrr Grrr", "HrrHrHrr", "khaaaaa", "pizza pizzza", "you dumb", "banana peel", "hit meee"] # need locale optimization
```
Comment (mirrorcult) [inline Resources/Prototypes/Traits/disabilities.yml:39]: im not sure about this one mate

**Comment by mirrorcult** [comment]
cant say im a fan of this in concept

File: Resources/Prototypes/Traits/disabilities.yml (hunk 1 above)
Comment (Gnomee1337) [reply Resources/Prototypes/Traits/disabilities.yml:39]: Yes, i agree, but idk how to make it available for locale

**Comment by EmoGarbage404** [comment]
this seems in really poor taste.

**Comment by moonheart08** [comment]
This is in incredibly poor taste. No.

**Pull request closed without merging**